docs/reference-guide/streams/overview.md

```diff
@@ -140,3 +141,25 @@ the Kafka cluster. However, even though we cannot guarantee **exactly once**
 semantics, we tried to minimize the possibility of processing messages multiple
 times. This means committing the message offsets to the Kafka cluster happens
 right after the transaction is committed to the database.
+
+## Setting a stream offset
+
+For Kafka streams users can manually set the offset of the next consumed message 
+with a call to the query procedure `mg.kafka_set_stream_offset`
+
+```cypher
+CALL mg.kafka_set_stream_offset(stream_name, offset)
+```
+
+option|description|type|example|default
+:-:|:-:|:-:|:-:|:-:
+stream name|Name of the stream to set the offset|string|"my_stream"|/
+offset|Offset number to set|int|0|/
+
+An offset of `-1` denotes the start of the stream i.e, offset 0.
```

Comment:
```suggestion
An offset of `-1` denotes the  beginning of the stream, in other words earliest available message in the stream.
```

I think mentioning `0` here is a bit confusing, as the earliest offset is not known (not all messages has to be kept).

Reply:
Rephrased a little bit but plz let me know what you think!